Bonobo.Git.Server/Security/ADRoleProvider.cs

```diff
@@ -35,7 +35,8 @@ public string[] FindUsersInRole(string roleName, string usernameToMatch)
         {
             if (String.IsNullOrEmpty(usernameToMatch)) throw new ArgumentException("Value cannot be null or empty.", "usernameToMatch");
 
-            return String.IsNullOrEmpty(usernameToMatch) ? ADBackend.Instance.Roles[roleName].Members : ADBackend.Instance.Roles[roleName].Members.Where(x => x.Contains(usernameToMatch)).ToArray();
+            //return String.IsNullOrEmpty(usernameToMatch) ? ADBackend.Instance.Roles[roleName].Members : ADBackend.Instance.Roles[roleName].Members.Where(x => x.Contains(usernameToMatch)).ToArray();
+            return new string[0];
```

Comment:
I'm a bit lost on these changes. Now all these functions return "false". Won't that impact who is member of which group/role?